src/Components/Header.jsx

```diff
@@ -1,12 +1,20 @@
 import React from "react";
+import { Link } from "react-router-dom";
+
 import styles from '../Styles/header.module.scss'
 import { ReactComponent as Logo } from '../Assets/bridge-logo.svg'
 
-function Header() {
+function Header({backRoute}) {
```

Comment:
I think we'll be refactoring this when we get the back/previous page path stuff working with react router history, because we won't need to pass a path to the Header component